src/core/index.ts

```diff
@@ -411,9 +411,10 @@ export class DevAssist {
 						const [userRejected, result] = await this.executeCommandTool(command);
 						pushToolResult(result);
 						// Add result to messages
+						console.log(result);
 						this.userMessageContent.push({
 							type: "text",
-							text: `Executed command: ${command}`,
+							text: `Executed command: ${command}, Result: ${result}`,	
```

Comment:
We do not required to update userMessageContent because  pushToolResult() function doing the same. This will be fixed in next PR

LGTM